tests/Integration/inc/Addons/Cloudflare/CloudflareSubscriber/TestCase.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace WP_Rocket\Tests\Integration\inc\Addons\Cloudflare\CloudflareSubscriber;
+
+use ReflectionClass;
+use WPMedia\Cloudflare\Cloudflare;
+use WPMedia\PHPUnit\Integration\TestCase as BaseTestCase;
+
+abstract class TestCase extends BaseTestCase {
+	protected static $container;
+	protected static $subscriber;
+
+	// Original instances within CloudflareSubscriber. Don't touch. We want to retain and then restore when done.
+	private static $subscriber_cf;
+	private static $subscriber_options;
+	private static $cf_property;
+	private static $options_property;
+
+	// API Credentials.
+	protected static $api_credentials_config_file;
+	protected static $api_credentials = [
+		'api_key'  => '',
+		'email'    => '',
+		'zone_id'  => '',
+		'site_url' => '',
+	];
+
+	public static function setUpBeforeClass() {
+		parent::setUpBeforeClass();
+
+		self::$api_credentials_config_file = WP_ROCKET_PLUGIN_ROOT . '/tests/env/local/cloudflare.php';
+
+		self::$container = apply_filters( 'rocket_container', '' );
+		self::setApiCredentials();
+		self::resetTransients();
+		self::$subscriber = self::$container->get( 'cloudflare_subscriber' );
+
+		// Store original state. Why? We'll restore it before existing this class.
+		$class    = new ReflectionClass( self::$subscriber );
+		self::$cf_property = $class->getProperty( 'cloudflare' );
+		self::$cf_property->setAccessible( true );
+		self::$subscriber_cf = self::$cf_property->getValue( self::$subscriber );
+		self::$options_property            = $class->getProperty( 'options' );
```

Comment:
```suggestion
		self::$options_property = $class->getProperty( 'options' );
```

Reply:
Applied this after I renamed the folder. https://github.com/wp-media/wp-rocket/pull/2313/commits/027d1c830137452b046dc34c796f8a5a94ed8c76